articles/quickstart/spa/angular2/06-token-renewal.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Token Renewal
+description: This tutorial demonstrates how to add automatic access_token renewal to an Angular 2+ application with Auth0
+budicon: 448
+---
+
+<%= include('../../../_includes/_package', {
+  org: 'auth0-samples',
+  repo: 'auth0-angular-samples',
+  path: '05-Token-Renewal',
+  requirements: [
+    'Angular 2+'
+  ]
+}) %>
+
+## Introduction
+As a security measure, it is recommended that the lifetime of a user's `access_token` be kept short. When you create an API in the Auth0 dashboard, the default lifetime is 7200 seconds (2 hours). This lifetime is different from the **JWT Expiration (seconds)** field in the client.
```

Comment:
> This lifetime is different from the **JWT Expiration (seconds)** field in the client.

Let's put this in a note

::: note
**Note:** The `access_token` lifetime is controlled from the [APIs section](https://manage.auth0.com/#/apis), while the `id_token` lifetime is controlled from the [Client Settings](https://manage.auth0.com/#/clients). These two settings are independent of one another.
:::